StorageDataModel/COItem.m

```diff
@@ -447,6 +469,27 @@ - (void)removeValueForAttribute: (NSString*)anAttribute
 
 /** @taskunit convenience */
 
+- (void)setEntityName: (NSString *)entityName
+{
+	[self setValue: entityName
+	  forAttribute: kCOObjectEntityNameProperty
+	          type: kCOTypeString];
+}
+
+- (void)setPackageVersion:(int64_t)entityVersion
+{
+	[self setValue: @(entityVersion)
+	  forAttribute: kCOObjectPackageVersionProperty
+			  type: kCOTypeInt64];
+}
+
+- (void)setPackageName:(NSString *)packageName
+{
+	[self setValue: [packageName copy]
```

Comment:
If we copy the package name, we should probably do the same for the entity name. It doesn't really matter though.